.env.config

```diff
@@ -0,0 +1,3 @@
+DEVFILE_SOURCE_REPOS="Official"
+DEVFILE_ENDPOINTS="https://registry.devfile.io/index/all?icon=base64"
```

Comment:
Why not just have one environment variable for the registry hostnames, `DEVFILE_REGISTRY_HOSTS`, and append on `/index/all?icon=base64` and `/devfiles/` when needed?

Reply:
Fixed